frontend/src/components/FormButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {Text, TouchableOpacity, StyleSheet } from 'react-native';
+
+const FormButton = ({buttonTitle, ...rest}) => (
+    <TouchableOpacity style={styles.buttonContainer} {...rest}>
+        <Text style={styles.buttonText}>{buttonTitle}</Text>
+    </TouchableOpacity>
+);
+
+export default FormButton;
+
+const styles = StyleSheet.create({
```

Comment:
Is there any way that we can improve this so we don't have a styles const at the bottom - things might get complicated if we need to update one thing and need to change many places